vagrant/Install-on-Centos-7.sh

```diff
@@ -52,15 +52,16 @@
 # we assume this user is called nominatim and the installation will be in
 # /srv/nominatim. To create the user and directory run:
 #
-#     sudo useradd -d /srv/nominatim -s /bin/bash -m nominatim
+sudo mkdir -p /opt/nominatim       #DOCS:    sudo useradd -d /srv/nominatim -s /bin/bash -m nominatim
+sudo chown vagrant /opt/nominatim  #DOCS:
 #
 # You may find a more suitable location if you wish.
 #
 # To be able to copy and paste instructions from this manual, export
 # user name and home directory now like this:
 #
     export USERNAME=vagrant        #DOCS:    export USERNAME=nominatim
-    export USERHOME=/home/vagrant  #DOCS:    export USERHOME=/srv/nominatim
+    export USERHOME=/opt/nominatim #DOCS:    export USERHOME=/srv/nominatim
```

Comment:
`/srv/nominatim` works for me as well. It looks like the .so file needs to exist when the SELinux instructions are run. So we'd need to move the SELinux instruction to the end of the file.

Reply:
yes, got it working in /srv now